server/controllers/article.controller.js

```diff
@@ -149,22 +149,33 @@ class ArticleController {
      * @returns {object}
      * @memberof getAllArticles
     */
+    const { page } = req.params;
+    const limit = 10;
+    const offset = limit * (page - 1);
+    const order = [['createdAt', 'DESC']];
     const articles = await Article.findAndCountAll({
+      limit,
+      offset,
+      order,
       where: { published: true },
       include: [{
         model: User,
         as: 'author',
         attributes: ['firstname', 'lastname', 'bio', 'image']
```

Comment:
Include 'username' among the attributes, to cover all possibilities.

Reply:
Okay